src/lib/april-fools-text.js

```
@@ -0,0 +1,261 @@
+
+    // Looks blocks
+    'say %1 for %2 seconds': 'chat %1 for %2 seconds 💬',
+    'say %1': 'chat %1 💬',
+    'think %1 for %2 seconds': 'ponder %1 for %2 seconds 🤔',
+    'think %1': 'ponder %1 🤔',
+    'switch costume to %1': 'change outfit to %1 👗',
+    'next costume': 'next outfit 👔',
+    'switch backdrop to %1': 'change scene to %1 🎭',
+    'next backdrop': 'next scene 🎬',
+    'change %1 effect by %2': 'add %2 %1 magic ✨',
+    'set %1 effect to %2': 'make %1 effect %2 🎨',
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)